### DISCUSSION TITLE
Lab settings and presets

### DISCUSSION THREAD

**Discussion by Puttanamadonna** [gopro/labs#887] [General]
Hello, just discovering Gopro Labs settings....one thing is not clear to me...I have 3 presets on the gopro 11, when I scan a QRcode with labs settings, these settings are applied to the whole camera itself, so at all the presets? Or I can have different presets with different Labs settings at the same time? So switching from a preset to another preset will change also the Labs setting...

**Comment by dnewman-gpsw** [comment]
Presets only hold things like resolution, frame rate and EV setting, i.e. everything you see in the preset menu on camera.  If you scan a command like ```r4``` (res 4K) it will modify the current preset. mVx <- where x is the present ID, can be used to select a preset on some camera models.  All the four character command extensions like *BITR=150 or $EXPS=1, are global, they effect all presets and the entire camera behavior.  Labs everything used to be global, as Labs existed (internally at GoPro) before the cameras added presets.

**Comment by Puttanamadonna** [comment]
ok understand......thank you for the clarification...
So at the end what is more convenient:

a) Make a QR code with global Labs settings that affect everything and just change the preset from the camera when you need

or

b) Create many Qr code on the app, each one complete with protune,ev etc, and the global labs setting

**Reply by dnewman-gpsw** [reply]
All case specific. You simply have more control with Labs.
